vernac/synterp.ml

```diff
@@ -292,20 +292,20 @@ let synterp_require from export qidl =
   let locate (qid,_) =
     let open Loadpath in
     match locate_qualified_library ?root qid with
-    | Ok (dir,_) -> dir
-    | Error LibUnmappedDir -> raise (UnmappedLibrary (root, qid))
-    | Error LibNotFound -> raise (NotFoundLibrary (root, qid))
+    | Ok (dir,_) -> (qid.loc, dir)
```

Comment:
As you are manipulating a `CAst.t`, you could instead try to preserve the `Ast` attributes for the client downwards, use for example `CAst.map_with_loc`.

Not a big deal tho.

I would be good at some point to remove the internal raising there, and use `Result.t`.